src/App.tsx

```diff
@@ -1,22 +1,31 @@
 /**@jsx jsx */
 import { Global, jsx } from '@emotion/react'
-import React from 'react'
+import * as React from 'react'
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom'
 
 import LoadingSpinner from './components/loading-spinner'
-import routes from './route.config'
+import Home from './routes/Home'
+import NotFound from './routes/NotFound'
 import { globalStyles } from './styles/globalStyles'
 
+const Raids = React.lazy(() => import('./routes/raids'))
+const Raid = React.lazy(() => import('./routes/raid'))
+
+const AppRouter = () => (
+  <React.Suspense fallback={<LoadingSpinner />}>
+    <Switch>
+      <Route path="/raids" exact component={Raids} />
+      <Route path="/raids/:raidId" exact component={Raid} />
+      <Route path="/" exact component={Home} />
+      <Route path="*" component={NotFound} />
+    </Switch>
+  </React.Suspense>
+)
+
```

Comment:
👍🏻 I like this mainly because I prefer the explicitness

Reply:
same <3